brightlayer-ui/doc-it
# Add release notes for R25

Fixes BLUI-3511


## Changes proposed in this Pull Request
- Added release notes for R25
  - Note that the packages are subject to change as React Native releases... package versions are not final yet
- Point roadmap to R26

## Screenshot

<img width="787" alt="image" src="https://user-images.githubusercontent.com/8997218/194922558-3498b9ef-fda2-4d8f-842f-d02880bf0d63.png">
<img width="685" alt="image" src="https://user-images.githubusercontent.com/8997218/194922645-0aebd64b-8de9-473c-9517-1e5ffaf2b792.png">



## To Test
- `yarn start`
- check home page, where it says "Latest Updates"
- check http://localhost:3000/release-notes
